lib/commands/doctor/checks/free-space.js

```diff
@@ -0,0 +1,34 @@
+const sysinfo = require('systeminformation');
+const {SystemError} = require('../../../errors');
+
+const MB_IN_BYTES = 1048576;
+
+// one version of Ghost takes ~400mb of space currently, to be safe we're going to say 1gb
+const MIN_FREE_SPACE = 1024;
+
+async function checkFreeSpace(ctx) {
+    const dir = ctx.instance ? ctx.instance.dir : process.cwd();
```

Comment:
Can't wait to `ctx.instance?.dir ?? process.cwd()` in the future

Reply:
💯